Gemfile

```diff
@@ -26,21 +25,24 @@ gem 'sidekiq-cron', '~> 1.9.1'
 gem 'aws-sdk-s3', '~> 1.94', require: false
 gem 'image_processing', '~> 1.12'
 gem 'active_storage_validations', '~> 1.1.3'
+gem "puma", ">= 5.0.0"
+gem 'matrix', '~> 0.4.1'
 
 # Assets
 gem 'jquery-rails', '~> 4.4.0'
 gem 'bootstrap-sass', '~> 3.4'
 gem 'sassc-rails', '~> 2.1.2'
-gem 'uglifier', '~> 4.2.0'
 gem 'select2-rails', '~> 4.0.13'
 
+group :production do
+  # we are using an ExecJS runtime only on the precompilation phase
+  gem "uglifier", "~> 4.2.0", require: false
```

Comment:
I'm ok with this, but I think some parity between dev/pro is also nice 👌🏼

Reply:
I've done it this way to avoid installing node on the final docker image. In any case, Uglifier is not used in the development mode as assets are not compressed.